src/ripple/protocol/impl/STAmount.cpp

```diff
@@ -1013,23 +1065,11 @@ divide (STAmount const& num, STAmount const& den, Issue const& issue)
         }
     }
 
-    // Compute (numerator * 10^17) / denominator
-    CBigNum v;
-
-    if ((BN_add_word64 (&v, numVal) != 1) ||
-            (BN_mul_word64 (&v, tenTo17) != 1) ||
-            (BN_div_word64 (&v, denVal) != 1))
-    {
-        Throw<std::runtime_error> ("internal bn error");
-    }
-
-    // 10^16 <= quotient <= 10^18
-    assert (BN_num_bytes (&v) <= 64);
-
-    // TODO(tom): where do 5 and 17 come from?
-    return STAmount (issue, v.getuint64 () + 5,
-                     numOffset - denOffset - 17,
-                     num.negative() != den.negative());
+    // mantissa: (numVal * 10^17) / denVal
```

Comment:
Personally, I think it would be a good idea to expand the comment.  The comment notes that we're using the magic number 10^17, but it doesn't explain *why*.  The comment also entirely slides past the other two magic numbers: 5 and 17.  Mysterious stuff.